database/migration/src/main/resources/changesets/call_caching_job_detritus.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+
+    <changeSet author="rmunshi" id="call_caching_job_detritus">
+        <comment>
+            One row per standard output file per job (i.e, one row for the stdout, stderr, etc)
```

Comment:
"standard output file" is kind of confusing when this encompasses a file of standard output as well as some other "detritus" files